Source/Core/Core/PowerPC/JitArm64/JitArm64_LoadStore.cpp

```diff
@@ -939,7 +940,7 @@ void JitArm64::dcbx(UGeckoInstruction inst)
   SwitchToNearCode();
   SetJumpTarget(near_addr);
 
-  gpr.Unlock(WA, WB, physical_addr);
+  gpr.Unlock(WA, WB);
   if (make_loop)
     gpr.Unlock(loop_counter);
```

Comment:
It would be nice to have a function in the register cache that lets us use scoped registers for fixed registers like these. But I suppose this PR is getting a bit large already, so you don't have to put that in this PR.

Reply:
Good idea, should be easy. I'll make a PR for that once this lands.